packages/rocketchat-lib/server/lib/PushNotification.js

```diff
@@ -0,0 +1,51 @@
+/* globals Push */
+class PushNotification {
+	getNotificationId(roomId) {
+		let serverId = RocketChat.settings.get('uniqueID');
+		return this.hash(`${serverId}|${roomId}`); // hash
+	}
+
+	hash(str) {
+		let hash = 0,
+			i = str.length;
+
+		while (i) {
+			hash = ((hash << 5) - hash) + str.charCodeAt(--i);
+			hash = hash & hash; // Convert to 32bit integer
+		}
+		return hash;
+	}
+
+	send({ roomName, roomId, username, message, usersTo, payload }) {
+		let title;
+		if (roomName && roomName !== '') {
+			title = `${roomName}`;
+			message = `${username}: ${message}`;
+		} else {
+			title = `${username}`;
+		}
+		let icon = RocketChat.settings.get('Assets_favicon_192').url || RocketChat.settings.get('Assets_favicon_192').defaultUrl;
+		const config = {
+			from: 'push',
+			badge: 1,
+			sound: 'chime',
```

Comment:
if we add our own sounds to the mobile apps, we can change the name of the here ;)